frontend/src/packages/chat-messages/libs/types/chat-message-create-payload.type.ts

```diff
@@ -0,0 +1,11 @@
+import {
+  type ChatMessageCreateRequestDto,
+  type ChatMessagesUrlParameters,
+} from './types.js';
+
+type ChatMessageCreatePayload = {
+  body: ChatMessageCreateRequestDto;
+  options: ChatMessagesUrlParameters;
```

Comment:
remove ChatMessagesUrlParameters
```suggestion
  message: string;
  chatId: number;
```